### DISCUSSION TITLE
Async vs sync

### DISCUSSION THREAD

**Discussion by Mayb3Nots** [isar/isar#316] [Help] [answered]
What's the difference between async and sync transition writes and reads? Is there a performance difference? Are there specific use cases for each? Thanks for the awesome package.

**Comment by simc** [comment, marked as answer]
Async operations run in a separate thread and don't block your UI but are a tiny bit slower. The difference is very similar to the sync and async operations of the Dart `File` API.

**Reply by Mayb3Nots** [reply]
I see, so we are trading performance for speed if we choose async and vice versa.
